bats/tests/k8s/wordpress.bats

```diff
@@ -1,9 +1,8 @@
-setup() {
-    load '../helpers/load'
-    if is_macos arm64; then
-        skip "The bitnami wordpress image is not available for arm64 architecture. Skipping..."
-    fi
-}
+load '../helpers/load'
+
+if is_macos arm64; then
+    fatal "The bitnami wordpress image is not available for arm64 architecture. Skipping..."
+fi
```

Comment:
This doesn't skip the test, it fails it instead.
```
 ✗ setup_file failed
   The bitnami wordpress image is not available for arm64 architecture. Skipping...
   (from function `teardown_file' in test file tests/k8s/../helpers/load.bash, line 38)
     `teardown_file() {' failed with status 0
```

Reply:
Indeed. I'm going to revert this. Unfortunately there doesn't seem to be a way to skip the whole file, but I guess that is fine:

```console
$ ./bats-core/bin/bats tests/k8s/wordpress.bats
wordpress.bats
 - factory reset (skipped: The bitnami wordpress image is not available for arm64 architecture. Skipping...)
 - add helm repo (skipped: The bitnami wordpress image is not available for arm64 architecture. Skipping...)
 - start rancher desktop (skipped: The bitnami wordpress image is not available for arm64 architecture. Skipping...)
 - deploy wordpress (skipped: The bitnami wordpress image is not available for arm64 architecture. Skipping...)
 - verify wordpress was deployed (skipped: The bitnami wordpress image is not available for arm64 architecture. Skipping...)

5 tests, 0 failures, 5 skipped
```